lang_python.adoc

```diff
@@ -13,98 +13,118 @@ To generate Python modules, Kaitai Struct compiler should be invoked with
 `-t python` option:
 
 [source,shell]
-kaitai-struct-compiler -t python my_spec.ksy
+kaitai-struct-compiler -t python [--python-package .] my_spec.ksy
 
 This normally generates `my_spec.py` parser module, which is compatible
-with both Python 2 and 3.
+with both Python 2 and 3. It is strongly recommended to use python 3
+because python 2 is EOL since 2020-01-01.
+
+`--python-package` optional CLI arg allows you to select a parent package
```

Comment:
Can you please add the code block with the Python statement it gets translated into? And does it get inserted at all times, or only when `meta/imports` in the `.ksy` spec are used?

I think people familiar with Python can better understand this CLI option when they see the Python equivalent, rather than when they have to read such a long paragraph of continuous text, which I didn't quite understand either. I've already suggested this in the second comment of https://github.com/kaitai-io/kaitai_struct_doc/pull/32#pullrequestreview-459605007. 

Every time when there's a opportunity to make the text shorter and more concise, e.g. explain the same thing with some code snippets or pictures imstead, it's a good idea to do it.

Reply:
I have added the example. But it hasn't made the text shorter, because some rationale to do so is still needed. It is obvious, but there are too many people for whom it is not.

Reply:
> It is obvious, but there are too many people for whom it is not.

It may be obvious to you, but it wasn't to me until you added the example. Now it's much better than before 👍